internal/pkg/aws/ecs/ecs.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package ecs contains utility functions for dealing with ECS repos
+package ecs
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ecs"
+)
+
+type ecsClient interface {
+	DescribeTaskDefinition(input *ecs.DescribeTaskDefinitionInput) (*ecs.DescribeTaskDefinitionOutput, error)
+}
+
+// Service wraps an AWS ECS client.
+type Service struct {
+	ecs ecsClient
+}
+
+// New returns a Service configured against the input session.
+func New(s *session.Session) *Service {
+	return &Service{
+		ecs: ecs.New(s),
+	}
+}
+
+// EnvVars calls ECS API and returns the environment variables of the task definition.
+func (s Service) EnvVars(taskDefName string) (map[string]string, error) {
+	resp, err := s.ecs.DescribeTaskDefinition(&ecs.DescribeTaskDefinitionInput{
+		TaskDefinition: aws.String(taskDefName),
+	})
+	if err != nil {
+		return nil, fmt.Errorf("describe task definition %s: %w", taskDefName, err)
+	}
+	envs := make(map[string]string)
+	for _, env := range resp.TaskDefinition.ContainerDefinitions[0].Environment {
```

Comment:
Question - do you think we should also include the values from `Secret`, and put _secret_ as the value? Or do you think we should have another section in app show.

Reply:
IMHO we should have another section if we want to include `Secret`. But I don't feel like we should add it by default because it would be a bit overwhelmed. How about adding it as an optional flag? Like resources?

Reply:
Hm, I think adding it would make sense, just because they're just special environment variables. We also show it in the manifest, so it might be worth echoing back. We can do it as a follow up, but I think it's worth showing.

Reply:
Oh makes sense to me :+1: